.github/workflows/release.yml

```diff
@@ -0,0 +1,87 @@
+      # Provisions Profiles and Certificates, used to deploy the iOS project to App Store Connect.
+      # Since this provisions and certificates are stored in an external repository, we must install the SSH private key
+      # that gives access to this repository, so we are able to clone it locally - and an easy and safe way to do it is
+      # by using SSH
+      - name: Install SSH key
+        uses: shimataro/ssh-key-action@v2
+        with:
+          key: ${{ secrets.SSH_PRIVATE_KEY }}
+          known_hosts: ${{ secrets.KNOWN_HOSTS }}
+          if_key_exists: replace
+
+      # Build an release iOS app to App Store Connect. More info about the steps in ios/fastlane/Fastfile file
+      - name: Release iOS App
+        run: cd ios && fastlane release --verbose
+        env:
+          APP_STORE_CONNECT_API_KEY: ${{ secrets.APP_STORE_CONNECT_API_KEY }}
+          APP_STORE_CONNECT_API_KEY_ID: ${{ secrets.APP_STORE_CONNECT_API_KEY_ID }}
+          APP_STORE_CONNECT_API_ISSUER_ID: ${{ secrets.APP_STORE_CONNECT_API_ISSUER_ID }}
+          APPLE_ID: ${{ secrets.APPLE_ID }}
+          ITC_TEAM_ID: ${{ secrets.ITC_TEAM_ID }}
+          TEAM_ID: ${{ secrets.TEAM_ID }}
+          KEYCHAIN_NAME: ${{ secrets.KEYCHAIN_NAME }}
+          KEYCHAIN_PASSWORD: ${{ secrets.KEYCHAIN_PASSWORD }}
+          PROVISIONS_CERTIFICATES_REPO: ${{ secrets.PROVISIONS_CERTIFICATES_REPO }}
+          MATCH_PASSWORD: ${{ secrets.MATCH_PASSWORD }}
+
+      # Retrieves keystore.jks and key.properties files from Github secrets. Both are used to sign the Android app
+      # before publishing it to Google Play console
+      - name: Setup Android Dependencies
+        run: |
+          echo "${{ secrets.KEYSTORE_BASE64_ENCODED }}" > keystore.jks.asc
+          gpg -d --passphrase "${{ secrets.KEYSTORE_PASSPHRASE }}" --batch keystore.jks.asc > android/app/keystore.jks
+          echo "${{ secrets.KEY_PROPERTIES_BASE64_ENCODED }}" > key.properties.asc
+          gpg -d --passphrase "${{ secrets.KEY_PROPERTIES_PASSPHRASE }}" --batch key.properties.asc > android/key.properties
+
+      # Build an release Android app to Google Play Console. More info about the steps in android/fastlane/Fastfile file
+      - name: Release Android App
+        run: cd android && fastlane internal --verbose
+        env:
+          GOOGLE_SERVICE_ACCOUNT: ${{ secrets.GOOGLE_SERVICE_ACCOUNT }}
+
+      # Commit versioning files updates to the repository
+      - name: Commit and push versions bump
+        run: |
+          git config --global user.name 'Memo Bot'
+          git config --global user.email 'memo@olmps.co'
+          git commit -am "[Fastlane] Bump version"
+          git push
+
+      # Run tests and upload code coverage do Codecov
+      - name: Run tests and upload coverage to Codecov
+        run: |
+          flutter test --coverage
+          curl -s https://codecov.io/bash | bash
```

Comment:
don't mix codecov with our private stuff, if you have ran this, we could have possibly leaked the keys already. Read more #39

Reply:
I removed the Codecov phase until we decide how to proceed with this security issues they are facing. Further discussions will be made through https://github.com/olmps/memo/issues/39